packages/core/src/types/oas3_1.ts

```diff
@@ -134,6 +134,7 @@ const Schema: NodeType = {
     then: 'Schema',
     else: 'Schema',
     dependentSchemas: listOf('Schema'),
+    dependentRequired: mapOf('DependentRequired'),
```

Comment:
No need in mapOf, as there are already additionalProperties in the DependentRequired:

```suggestion
    dependentRequired: 'DependentRequired',
```